Assets/Scripts/HighlightManager.cs

```
@@ -63,18 +63,37 @@ public void ProcessMessage(Message message)
             _activeHighlightCount = Math.Min(highlightsMessage.Length, _highlightPool.Length);
             for (int i = 0; i < _activeHighlightCount; ++i)
             {
-                var highlight = _highlightPool[i];
-                highlight.enabled = true;
+                Highlight msg = highlightsMessage[i];
+                var lineRenderer = _highlightPool[i];
+                lineRenderer.enabled = true;
+
+                Color color = _config.highlightDefaultColor;
+                if (msg.c != null && msg.c.Length == 4)
```

Not sure if silent failure is the best option here if Length != 4.